cas-server-support-jpa-util/src/main/java/org/apereo/cas/util/jpa/ZonedDateTimeConverter.java

```diff
@@ -9,20 +9,29 @@
 
 /**
  * Converts ZonedDateTime &lt;-&gt; Date.
+ *
  * @author Timur Duehr timur.duehr@nccgroup.trust
  * @since 5.0.0
  */
 @Converter(autoApply = true)
 public class ZonedDateTimeConverter implements AttributeConverter<ZonedDateTime, Date> {
 
     @Override
-    public Date convertToDatabaseColumn(final ZonedDateTime zonedDateTime){
-        return DateTimeUtils.dateOf(zonedDateTime);
+    public Date convertToDatabaseColumn(final ZonedDateTime zonedDateTime) {
+        if (zonedDateTime != null) {
```

Comment:
We need ZonedDateTimeConverter work to convert ZonedDateTime to java SQL type，we want the database column type is timestamp，not oid。